reviews/views.py

```diff
@@ -1,3 +1,68 @@
-from django.shortcuts import render
+import boto3
+import json
+import uuid
 
-# Create your views here.
+from django.http  import JsonResponse
+from django.views import View
+
+from core.utils     import login_required
+from my_settings    import AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY
+from arts.models    import Art
+from reviews.models import Review, ReviewImage
+
+class ReviewView(View):
+    @login_required
+    def post(self, request, art_id):
+        try:
+            user = request.user
+            art  = Art.objects.get(id=art_id)
+
+            review = Review.objects.create(
+                user_id = user.id,
+                art_id  = art.id,
+                comment = request.POST.get("comment")
+            )
```

Comment:
```suggestion
            if not Art.objects.filter(id=art_id).exists():
                return JsonResponse({"message" : "NOT_FOUND"}, status=404)

            review = Review.objects.create(
                user_id = user.id,
                art_id  = art_id,
                comment = request.POST.get("comment")
            )
```
art_id의 존재 유무를 먼저 판단해야 합니다. 그래서 만약 id가 존재하지 않으면, 에러를 보내주어야 합니다. 그리고 존재 할 경우는 이미 id를 알고 있으므로 get을 통해 객체를 불러올 필요없이 바로 id를 활용해 주면 될 것 같습니다.

Reply:
filter().exists() 메소드를 사용하는 것으로 수정했습니다, 감사합니다!